Classes/Extensions/DeclarativeProtocol+Apply.swift

```diff
@@ -0,0 +1,13 @@
+//
+// Created by Kevin Guo on 2022/1/3.
+//
+
+import Foundation
+extension DeclarativeProtocol {
```

Comment:
Also would be cool to add some little comment which would explain for what this method can be used for e.g.

```swift
/// Helper method where you could change the object non-declarative way
///
/// ```swift
/// view1.apply {
///     $0.backgroundColor = .red
/// }
/// ```
@discardableResult
public func apply(_ closure: (V) -> Void) -> Self {
    /// ...
}
```
So users will see the hint and example during autocompletion or when they left-click on it with `Option`.